include/bluetooth/bluetooth.h

```diff
@@ -398,6 +525,140 @@ int bt_le_adv_update_data(const struct bt_data *ad, size_t ad_len,
  */
 int bt_le_adv_stop(void);
 
+/** @brief Create advertising set.
+ *
+ *  Create a new advertising set and set advertising parameters.
+ *
+ *  @param param Advertising parameters.
+ *
+ *  @return Zero on success or (negative) error code otherwise.
+ *  @return -ECONNREFUSED When connectable advertising is requested and there
```

Comment:
This API returns a pointer so -ECONNREFUSED makes no sense. I guess you mean NULL? Since we don't have Linux-style `PTR_ERR()` and `ERR_PTR()` macros, if you really need the specific error you might need to add a `int *err` parameter to the function.

Reply:
Copy-paste bug here. Updated return value documentation.